dali/operators/audio/resample.cc

```diff
@@ -96,6 +96,17 @@ type. Example::
 )",
     nullptr, false);
 
+// Deprecated alias
+DALI_SCHEMA(experimental__AudioResample)
+    .AddParent("AudioResample")
+    .NumInput(1)
+    .NumOutput(1)
```

Comment:
I suggest to use the readers treatment and remove most of the documentation:
```suggestion
    .NumOutput(1)
    .MakeDocPartiallyHidden()
```